README.md

```diff
@@ -161,15 +161,35 @@ First build the image inside the project's root directory:
 $ docker build -t flathunter .
 ```
 
-**When running a container using the image, a config file needs to be mounted on the container at ```/config.yaml```.** The example below provides the file ```config.yaml``` off the current working directory:
+**When running a container using the image, a config file needs to be mounted on the container at ```/config.yaml``` or configuration has to be supplied using environment variables.** The example below provides the file ```config.yaml``` off the current working directory:
```

Comment:
We should think about separating the README into separate .md-files and only use the main file as index, because it is so large already

Reply:
Yeah. That would make sense. If you have a proposal for how you would like to see that split, I would welcome that. At least the basic getting-started instructions should maybe be at the top of the main README, and people who want to know more can dig a little.